src/components/Encounter/CreateEncounterForm.tsx

```diff
@@ -275,7 +275,9 @@ export default function CreateEncounterForm({
                     >
                       <FormControl>
                         <SelectTrigger ref={field.ref}>
-                          <SelectValue placeholder="Select status" />
+                          <SelectValue
+                            placeholder={t("fasting_status_placeholder")}
```

Comment:
```suggestion
                            placeholder={t("select_status")}
```

let's use `select_status` instead of `fasting_status_placeholder` 

Remove the  `fasting_status_placeholder`  from en.json and update the places where it is used as well with `select_status`

https://github.com/ohcnetwork/care_fe/blob/d381e5becea8ccd540141ae15a7764c99553f7ab/public/locale/en.json#L4817

Reply:
Hi Yash (@yash-learner),
Thanks for reviewing my PR. I’ve made the requested changes.
Could you please review it again when you have time?